docs/architecture/gas/README.md

```diff
@@ -1,27 +1,125 @@
-concepts](https://docs.near.org/concepts/basics/transactions/gas-advanced), and
-[the runtime fee specification](https://nomicon.io/RuntimeSpec/Fees/).
-
-So-called gas cost parameters are part of the protocol definition which can
-change between versions. The section on [Parameter Definitions](./parameter_definition.md)
-explains how to find the source of truth for such parameter's values in the
-nearcore repository and how they can be referenced in code.
-
-The [Gas Profile](./gas_profile.md) section goes into more details on how gas
-costs of a transaction are tracked in nearcore.
+Gas in NEAR Protocol solves two problems.
+1. To avoid spam, validator nodes only perform work if a user's tokens are
+   burned. Tokens are automatically converted to gas using the current gas
+   price.
+2. To synchronize shards, they must all produce chunks following a strict
+   schedule of 1 second execution time. Gas is used to measure how heavy the
+   workload of a transaction is, so that the number of transactions that fit in
+   a block can be deterministically computed by all nodes.
+
+In other words, each transaction costs a fixed amount of gas. This gas cost
+determines how much a user has to pay and how much time nearcore has to execute
+the transaction.
+
+What happens if nearcore executes a transaction too slowly? Chunk production for
+the shard gets delayed, which delays block production for the entire blockchain,
+reducing latency and throughput for everybody. If the chunk is really late, the
+block producer will decide to not include the chunk at all and inserts an empty
+chunk. The chunk may be included in the next block.
+
+By now, you probably wonder how we can know the time it takes to execute a
+transaction, given that validators use hardware of their choice. Getting these
+timings right is indeed a difficult problem. Or flipping the problem, assuming
+the timings are already known, then we must implement nearcore such that it
+guarantees to operate within the given time constraints. How we tackle this is
+the topic of this chapter.
+
+If you want to learn more about Gas from a user perspective, 
+[Gas basic concepts](https://docs.near.org/concepts/basics/transactions/gas),
+[Gas advanced concepts](https://docs.near.org/concepts/basics/transactions/gas-advanced),
+and [the runtime fee specification](https://nomicon.io/RuntimeSpec/Fees/) are
+good places to dig deeper.
+
+## Hardware and Timing Assumptions
+
+For timing to make sense at all, we must first define hardware constraints. The
+official hardware requirements for a validator is published on
+[near-nodes.io/validator/hardware](https://near-nodes.io/validator/hardware). It
+may change over time but the main principle is that a moderately configured,
+cloud-hosted virtual machine suffices.
+
+For our gas computation, we assume the minimum required hardware. Then we define
+10¹⁵ gas to be executed in at most 1s. We commonly use 1 Tgas (= 10¹² gas) in
```

Comment:
`10¹⁵` looks fine in the rendered form, but in the raw form theer's a double superscript.

Reply:
replaced with `<sup>15</sup>`